apis/v1beta1/validation/httproute.go

```diff
@@ -334,6 +341,28 @@ func validateHTTPHeaderModifier(filter gatewayv1b1.HTTPHeaderFilter, path *field
 	return errs
 }
 
+func validateHTTPRouteTimeouts(timeouts *gatewayv1b1.HTTPRouteTimeouts, path *field.Path) field.ErrorList {
+	var errs field.ErrorList
+	errorString := fmt.Sprintf("timeout value must be greater than or equal to %d ms", timeoutMinValue)
+	if timeouts.BackendRequest != nil {
+		if int(timeouts.BackendRequest.Duration.Milliseconds()) < timeoutMinValue {
+			errs = append(errs, field.Invalid(path.Child("backendRequest"), timeouts.BackendRequest.Duration, errorString))
+		}
+		if timeouts.Request != nil {
+			if int(timeouts.BackendRequest.Duration.Milliseconds()) > int(timeouts.Request.Duration.Milliseconds()) {
+				errs = append(errs, field.Invalid(path.Child("backendRequest"), timeouts.BackendRequest.Duration, "backendRequest timeout cannot be longer than request timeout"))
+			}
+		}
+	}
+	if timeouts.Request != nil {
+		if int(timeouts.Request.Duration.Milliseconds()) < timeoutMinValue {
```

Comment:
```suggestion
		timeout := int(timeouts.Request.Duration.Milliseconds())
		if timeout < timeoutMinValue && timeout != 0 {
```

Reply:
addressed in 4056411a2ddccafe6f9eb40bb9b14fd685e1859e